core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1355,11 +1389,10 @@ private String buildInOperator(int size) {
         if (size == 1) {
             return "(?)";
         }
-        StringBuilder builder = new StringBuilder("(");
-        for (int i = 0; i < size - 1; i++) {
-            builder.append("?,");
-        }
-        builder.append("?)");
+        StringBuilder builder =
+                new StringBuilder("(")
+                        .append(StringUtils.join(Collections.nCopies(size, "?"), ","))
```

Comment:
I am only aware of this issue because we already ran into it a couple of times ;)

Thought about it a couple of times already, yes. But then I bump into someone still using Android 2.3.7...
I sometimes regret not being able to use a library, but it hasn't been that big of an issue so far supporting sdk 10. But if you have something nice in mind...